api/libretime_api/permissions.py

```diff
@@ -98,10 +115,12 @@ def has_permission(self, request, view):
             if perm == "view_apiroot":
                 return True
             return request.user.has_perm(perm)
-        return check_authorization_header(request)
+
+        return super().has_permission(request, view)
 
     def has_object_permission(self, request, view, obj):
         if request.user and request.user.is_authenticated:
             perm = get_permission_for_view(request, view)
             return request.user.has_perm(perm, obj)
-        return check_authorization_header(request)
+
+        return super().has_object_permission(request, view, obj)
```

Comment:
I am a bit lost with our custom permission management system. IS there a way to simplify it ?

I also wonder if we currently expose way too much data to the public, shouldn't we only allow the api key auth and admin auth for now and slowly grant permissions when needed ?

Reply:
Currently this mimics the legacy permissions - ensuring that users have access to things via the API that they would have access to via the UI. Nothing sensitive should be publicly accessible